src/components/06-accordion/accordion.njk

```diff
@@ -1,16 +1,17 @@
-<ul class="usa-accordion{%
+<div class="usa-accordion{%
   if accordion.variant %}-{{ accordion.variant }}{% endif %}"{%
   if accordion.multiselectable == true %} aria-multiselectable="true"{% endif %}>
   {% for item in accordion.items %}
-  <li>
+  <!-- Use the accurate heading level to maintain the document outline -->
+  <h2 class="usa-accordion-heading">
     <button class="usa-accordion-button"
       aria-expanded="{{ item.expanded | default('false') }}"
       aria-controls="{{ accordion.id_prefix }}{{ item.id }}">
       {{ item.title }}
     </button>
-    <div id="{{ accordion.id_prefix }}{{ item.id }}" class="usa-accordion-content">
-      {{ item.content }}
-    </div>
-  </li>
+  </h2>
+  <div id="{{ accordion.id_prefix }}{{ item.id }}" class="usa-accordion-content">
+    {{ item.content }}
```

Comment:
In an effort to keep HTML tags out of the config we should probably move the `<p>` here to wrap {{ item.content }}

Reply:
I'm testing other text elements (e.g. headings and lists) to ensure resilience and flexibility in the accordion panels, so keeping the text in the config for now.